bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -28,14 +29,14 @@ const propTypes = {
   subscriptionsReady: PropTypes.bool.isRequired,
   locale: PropTypes.string,
   approved: PropTypes.bool,
-  meetingEnded: PropTypes.bool,
+  meetingIsEnded: PropTypes.bool,
   meetingExist: PropTypes.bool,
 };
 
 const defaultProps = {
   locale: undefined,
   approved: undefined,
-  meetingEnded: false,
+  meetingIsEnded: false,
```

Comment:
I would think that this prop could be `.isRequired` especially since we set it to `false` on server side when adding meeting